src/encoded/schemas/experiment.json

```diff
@@ -143,11 +143,19 @@
                         }
                     }
                 },
+                {
+                    "required": ["date_submitted"],
+                    "properties": {
+                        "status": {
+                            "enum": ["submitted"]
+                        }
+                    }
+                },
                 {
                     "not": {
                         "properties": {
                             "status": {
-                                "enum": ["released", "revoked"]
+                                "enum": ["released", "revoked", "submitted"]
```

Comment:
What happens when someone goes from in progress -> released, skipping submitted? This seems to allow released files with no date_submitted.

Reply:
the dependency we add now will allow your scenario. We have to backfill date submitted to many released datasets, before we ad dependency that required date_submitted to released datasets as well.